### PR TITLE
Handholding

### PR DESCRIPTION
**Pull request by Swagile**
## About The Pull Request

This is my meme PR for the month.

## Why It's Good For The Game

You thought I was joking, Tivi? Think again.

## Changelog
:cl:
add: Adds handholding. Now you can hold your SL's hand before you get atomized by GOOD HITS. Help intent and aim at either hand (not arm). Locked behind fun config.
/:cl:

### CODE REVIEW

**Comment by David-Stormwell** [comment]
what have we done to this server

**Comment by Swagile** [comment]
> 
> 
> what have we done to this server

you got your chance with queen charge

now suffer the wraith of ... marine hand holding

**Comment by XSlayer300** [comment]
dangit stop adding what i removed >:(

**Comment by Swagile** [comment]
> 
> 
> dangit stop adding what i removed >:(

the only one who can stop me

is tivi

but will he dare?

tune in next time for Xeno Ball TGMC

**Review by SplinterGP — COMMENTED** [review]
lock to fun config

File: code/modules/mob/living/carbon/human/species.dm [hunk 1]
```diff
@@ -165,7 +165,13 @@
...
 		H.visible_message("<span class='notice'>[H] pats [target] on the head.</span>", \
 					"<span class='notice'>You pat [target] on the head.</span>", null, 4)
-	else
+	if(H.zone_selected == "l_hand")
+		H.visible_message("<span class='notice'>[H] holds [target] 's left hand.</span>", \
+					"<span class='notice'>You hold [target]'s left hand.</span>", null, 4)
+	if (H.zone_selected == "r_hand")
+		H.visible_message("<span class='notice'>[H] holds [target] 's right hand.</span>", \
+					"<span class='notice'>You hold [target]'s right hand.</span>", null, 4)
+	if(H.zone_selected == "chest")
```
Comment (SplinterGP) [inline code/modules/mob/living/carbon/human/species.dm:174]:
```suggestion
	else
```

**Review by SplinterGP — CHANGES_REQUESTED** [review]

**Comment by Swagile** [comment]
dunno how to fiddle with configs on local server, but when i tested, i couldn't hand hold again but no runtimes happened, so i presume the fun flag code i did is working

**Review by SplinterGP — APPROVED** [review]

**Review by TiviPlus — APPROVED** [review]

**Comment by AnCopper** [comment]
degenerate...

**Pull request merged**